docs/settings.rst

```diff
@@ -238,6 +204,48 @@ produces in its output channel. Set to one of "trace", "debug", "info", "note",
     to the CMake Tools log file. This file is useful for
     :ref:`troubleshooting <troubleshooting>`.
 
+.. _conf-cmake.mingwSearchDirs:
+
+``cmake.mingwSearchDirs``
+*************************
+
+List of paths to search for a MinGW installation. This means that GCC does not
+need to be on your ``$PATH`` for it to be found via Kit scanning.
+
+- Default: ``["C:\\MinGW"]`` (Search in C:\\MinGW for a MinGW installation)
+
+.. _conf-cmake.parallelJobs:
+
+``cmake.parallelJobs``
+**********************
+
+By specifying a number, you can define how many jobs are run in parallel during the build.
```

Comment:
I would probably say that this controls how many CPU cores are used during the build.  `-j` is passed to build tools that support it, but not all do.  I believe msbuild uses `/maxcpucount:`.  I think if you just say that this controls the number of CPU cores used during the build, that should be sufficient.

Reply:
You're right. I changed it.